### PR TITLE
If the user is only manager, staff permission is not enough to display that page

### PR DESCRIPTION
**Pull request by Gustry**
If the user is only manager, staff permission is not enough to display that page

### CODE REVIEW

File: django_project/lesson/views/question.py [hunk 1]
```diff
@@ -126,8 +125,7 @@ def get_success_url(self):
         })
 
 
-class QuestionOrderView(
-    StaffuserRequiredMixin, QuestionMixin, ListView):
+class QuestionOrderView(LoginRequiredMixin, QuestionMixin, ListView):
     """List view to order questions"""
```
Comment (sonlinux) [inline django_project/lesson/views/question.py]: The code looks okay,
Lets add a dot **.** to end the docstring

File: django_project/lesson/views/section.py [hunk 2]
```diff
@@ -175,7 +187,7 @@ def get_success_url(self):
         return url
 
 
-class SectionOrderView(StaffuserRequiredMixin, SectionMixin, ListView):
+class SectionOrderView(LoginRequiredMixin, SectionMixin, ListView):
     """List view to order section"""
```
Comment (sonlinux) [inline django_project/lesson/views/section.py]: Same here a **.** in the docstring

File: django_project/lesson/views/specification.py [hunk 3]
```diff
@@ -127,7 +127,7 @@ def get_success_url(self):
 
 
 class SpecificationOrderView(
-    StaffuserRequiredMixin, SpecificationMixin, ListView):
+    LoginRequiredMixin, SpecificationMixin, ListView):
     """List view to order specifications"""
```
Comment (sonlinux) [inline django_project/lesson/views/specification.py]: Same here a **.** in the docstring

File: django_project/lesson/views/question.py (hunk 1 above)
Comment (Gustry) [reply django_project/lesson/views/question.py]: I will fix them in one go after this PR with a regex ;-)
Thanks

**Comment by ann26** [comment]
Please fix the travis error :) otherwise LGTM

**Pull request merged**